twoliter/embedded/Makefile.toml

```diff
@@ -894,6 +894,29 @@ ln -snf "${BUILDSYS_VERSION_FULL}" "${OUTPUT_LOGS_DIR}/latest"
 '''
 ]
 
+[tasks.build-all]
+dependencies = ["fetch", "build-sbkeys", "publish-setup", "cargo-metadata"]
```

Comment:
Either this PR or https://github.com/bottlerocket-os/twoliter/pull/361 will need to rebase here. I have no problem doing so, and I think you have your approvals first!